inference-engine/src/plugin_api/cpp_interfaces/interface/ie_iplugin_internal.hpp

```diff
@@ -166,6 +166,21 @@ class IInferencePlugin : public std::enable_shared_from_this<IInferencePlugin> {
     virtual std::shared_ptr<IExecutableNetworkInternal> LoadNetwork(const CNNNetwork& network,
                                                                     const std::map<std::string, std::string>& config,
                                                                     RemoteContext::Ptr context) = 0;
+
+    /**
+     * @brief Creates an executable network from model file path
+     * @param modelPath A path to model
+     * @param config A string-string map of config parameters relevant only for this load operation
+     * @return Created Executable Network object
+     */
+    virtual ExecutableNetwork LoadNetwork(const std::string& modelPath,
```

Comment:
@ilya-lavrenov All other methods in this interface are pure-virtual. Is it ok to have virtual methods with default implementation here? Otherwise in case of pure-virtual way all plugins will need to implement this method manually

Reply:
Maybe we can move this to `ie_plugin_internal.hpp` which provide minimal implementation of `IInferencePlugin` interface?

Reply:
GNAPlugin and MockPlugin use IInferencePlugin directly. I can change them as well, but it is a risk that other plugins from other repositories will not be compiled due to pure virtual method. Are we ok to potentially break such 3rd party plugins?

Reply:
> Are we ok to potentially break such 3rd party plugins?

Yes.